react/src/pages/utils.tsx

```diff
@@ -116,6 +159,31 @@ export function toKeyValue(items: string[]) {
     }, Object.create(null));
 }
 
+/**
+ * Return property of object at key.
+ *
+ * @example
+ * let obj = { a: 1, b: 2 };
+ * let id = "a";
+ * getProp(obj, id)          // returns 1
+ */
+export function getProp<T, K extends keyof T>(obj: T, key: K): T[K] {
+    return obj[key];
+}
+
+/**
+ * Set property of object at key to newValue. Return obj.
+ *
+ * @example
+ * let obj = { a: 1, b: 2 };
+ * setProp(obj, "a", 3);     // returns { a: 3, b: 2 }
+ * obj.a === 3               // returns true
+ */
+export function setProp<T, K extends keyof T>(obj: T, key: K, newValue: any) {
+    obj[key] = newValue;
+    return obj;
+}
+
```

Comment:
Per previous comments, these don't seem to really be necessary unless they're really needed in some FP context